notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1133 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    gkf_split = gkf.split(frc_data, groups=frc_data["plant_id_eia"])
</pre><p>In light of the geographic clustering of missing data, I'm wondering if we might try doing a grouped split by <code>state</code> to test the spatial dependency issue. Which do you think is more representative of our actual missing data problem?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>So this would be grouping for a different reason than avoiding intra-group information leakage right? In this case we'd be checking whether or not there's enough regional correlation that we can hope to infer prices in states where we have little to no information, based on information from other states?</p><p> </p><p>So if we were to hold out whole states, and saw model performance go down substantially, then we'd be worried, since it would mean that maybe we can't effectively get at what's going on the NE and other IPP dominated areas.</p><p> </p><p>Looking at the aggregate EIA API data for the states that aren't well represented, and seeing what kinds of correlations it has with our predicted (and the observed) prices in adjacent states might also be helpful?</p>